src/schema/workflow_execution_activity.yaml

```diff
@@ -553,3 +561,14 @@ slots:
     range: MetaboliteIdentification
     multivalued: true
     inlined_as_list: true
+
+enums:
+  MetaproteomicsAnalysisCategoryEnum:
+    description: The category of metaproteomics analysis being performed.
+    permissible_values:
```

Comment:
the PV description are more value-added than the `MetaproteomicsAnalysisCategoryEnum` description!